modules/bootstrapped/test/src/smithy4s/builder/ServiceBuilderSpec.scala

```diff
@@ -0,0 +1,72 @@
+package smithy4s
+
+
+import munit._
+import smithy4s.kinds.PolyFunction5
+import smithy4s.example.FooServiceGen
+
+class ServiceBuilderSpec extends FunSuite {
+
+  val service = smithy4s.example.FooService
+
+  val builder =  smithy4s.Service.Builder.fromService(service)
+
+  test(
+    "can replace the following values (Id, Version and Hints) using withId, withVersion, withHints"
+  ) {
+
+    val newService = builder
+      .withId(ShapeId.apply("smithy4s.example", "newService"))
+      .withHints(Hints(Hints.Binding.DynamicBinding.apply(ShapeId.apply("smithy.api", "documentation"), Document.fromString("this is new Service"))))
```

Comment:
**issue:** same suggestion as in the endpoint test, you can pass hints with less boilerplate